CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Global code owners: co-chairs
+*       @ultrasaurus @dshaw @pragashj
+
+# Security assessments (TODO: Approval required by PR)
```

Comment:
I don't understand the `TODO` here

Reply:
The TODO here is because the names are currently commented out and we will do a separate PR to add these users to CODEOWNERs file so that they can acknowledge. The approval in the PR by the person being added to the list will be used as an acknowledgement of the responsibility.